perses/rjmc/geometry.py

```diff
@@ -455,11 +467,10 @@ def _bond_logp(self, r, bond, beta):
         """
         k_eq = bond.type.k*units.kilocalories_per_mole/(units.angstrom**2)
```

Comment:
AMBER bond force constants may be off by a factor of two. Check what @swails does here carefully?
https://github.com/ParmEd/ParmEd/blob/master/parmed/structure.py#L2002-L2003